File: TP1/colorsOC/colorsOC/ViewController.m
//
//  ViewController.m
//  colorsOC
//
//  Created by m2sar on 07/10/2020.
//  Copyright © 2020 Sorbonne Universite. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [[self button] setMaximumValue:4];
}


- (IBAction)action:(UIStepper *)sender {
    if([self button].value == 0){
        [[self cadre] setBackgroundColor:UIColor.redColor];
    }
    if([self button].value == 1){
        [[self cadre] setBackgroundColor:UIColor.brownColor];
    }
    if([self button].value == 2){
        [[self cadre] setBackgroundColor:UIColor.cyanColor];
    }
    if([self button].value == 3){
        [[self cadre] setBackgroundColor:UIColor.yellowColor];
    }
    if([self button].value == 4){
        [[self cadre] setBackgroundColor:UIColor.blueColor];
    }
}
@end

File: TP1/colorS/colorS/ViewController.swift
//
//  ViewController.swift
//  colorS
//
//  Created by Caroline Huynh on 14/10/2020.
//  Copyright © 2020 Sorbonne Universite. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var cadre: UIView!
    @IBOutlet weak var stepper: UIStepper!
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        stepper.maximumValue = 4
    }

    @IBAction func actionStepper(_ sender: Any) {
        if(stepper.value == 0.0){
            cadre.backgroundColor = .yellow
        }
        if(stepper.value == 1.0){
            cadre.backgroundColor = .red
        }
        if(stepper.value == 2.0){
            cadre.backgroundColor = .purple
        }
        if(stepper.value == 3.0){
            cadre.backgroundColor = .lightGray
        }
        if(stepper.value == 4.0){
            cadre.backgroundColor = .blue
        }
    
    }
    
}

